CONTRIBUTING.rst

```diff
@@ -200,6 +200,69 @@ can become the Mentor and guide the proposed candidates on how they can become a
     the mentor must try to remain impartial and cannot be the Proposer.
 
 
+Inactive Committers
+-------------------
+If you know you are not going to be able to contribute for a long time
+(for instance due to a change of job or circumstances) you should inform the PMC and we will mark you
+as "inactive". Inactive committers will be removed from the "roster" on ASF and will no longer have the power
+of being a Committer (especially write access to the repos). As merit earned never expires, once you
+become active again you can simply email the PMC and ask to be reinstated.
+
+The PMC also can mark committers as inactive after they have not been involved in the community for
+more than 12 months.
+
+
+Guidelines to become an Airflow Committer
```

Comment:
Is is related to "Committer Responsibilities" section?
https://github.com/apache/airflow/blob/6f78c4b909de875809825f18c1873e724bd20ce6/CONTRIBUTING.rst#committer-responsibilities

Reply:
I guess we might link to it. 

This is rather different level but they describe what "active" committer should be responsible for.